src/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+  margin: 0;
+  background-color: darkgray;
+  box-sizing: border-box;
```

Comment:
`box-sizing: border-box` is not inherited, if you want to apply this to all your elements, use *-selector